runtime/gc_base/IndexableObjectAllocationModel.cpp

```diff
@@ -396,3 +413,83 @@ MM_IndexableObjectAllocationModel::getSparseAddressAndDecommitLeaves(MM_Environm
 	return spine;
 }
 #endif /* defined(J9VM_GC_SPARSE_HEAP_ALLOCATION) */
+
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+#if !((defined(LINUX) || defined(OSX)) && defined(J9VM_ENV_DATA64))
+/* Double map is only supported on LINUX 64 bit Systems for now */
```

Comment:
Can you please correct this comment? It's inconsistent with line 418.

Reply:
I would not. This is to be pulled out again, soon (likely before next split).

Reply:
If by "pulled out" you mean that support will be expanded beyond Linux and macOS, then it's probably not worth fixing the comment.

Reply:
I think it means all double mapping support will be deleted (this PR reverted). Replace by off heap array support.